### PR TITLE
Enforce Python version 3.5 and above

### PR DESCRIPTION
**Pull request by danyeaw**
Signed-off-by: Dan Yeaw <dan@yeaw.me>

### CODE REVIEW

File: beekeeper.yml [hunk 1]
```diff
@@ -6,8 +6,8 @@ pull_request:
                 name: Python lint checks
                 task: beefore
     - smoke-test:
-        task: py34
-        name: Smoke build (Python 3.4)
+        task: py35
+        name: Smoke build (Python 3.5)
     - full-test:
         subtasks:
             - py3.5:
```
Comment (freakboy3742) [inline beekeeper.yml]: No need to duplicate 3.5 here.

File: README.rst [hunk 2]
```diff
@@ -5,7 +5,7 @@
 Travertino
 ==========
 
-.. image:: https://img.shields.io/pypi/pyversions/travertino.svg
```
Comment (freakboy3742) [inline README.rst]: What's the reason for this change?

File: beekeeper.yml (hunk 1 above)
Comment (danyeaw) [reply beekeeper.yml]: Ok, updated.

File: README.rst (hunk 2 above)
Comment (danyeaw) [reply README.rst]: I believe the original version shield was created using a selected one that is available on the shields.io website that lists versions 3.4, 3.5, 3.6. I had to create a new "static" shield that lists the updated versions we want to list (3.5, 3.6, 3.7).

Comment (freakboy3742) [reply README.rst]: This isn't needed anymore; 3.7 is now officially supported. (see https://img.shields.io/pypi/pyversions/briefcase.svg)

Comment (danyeaw) [reply README.rst]: Got it, sorry I didn't realize these shields were dynamically generated

**Comment by danyeaw** [comment]
@freakboy3742 any additional updates needed for this one? ♥
